DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -501,89 +508,111 @@
     process.siStripDigis.ProductLabel = cms.InputTag("rawDataRepacker")
     process.siStripFEDMonitor.RawDataTag = cms.untracked.InputTag("rawDataRepacker")
 
-    # Select events based on the pixel cluster multiplicity
-    import  HLTrigger.special.hltPixelActivityFilter_cfi
-    process.multFilter = HLTrigger.special.hltPixelActivityFilter_cfi.hltPixelActivityFilter.clone(
-        inputTag  = cms.InputTag('siPixelClusters'),
-        minClusters = cms.uint32(10000),
-        maxClusters = cms.uint32(50000)
-        )
-    # Trigger selection
-#    process.DQMEventStreamerReader.SelectEvents = cms.untracked.PSet(SelectEvents = cms.vstring('HLT_HIJet*','HLT_HICentralityVeto*','HLT_HIFullTrack*','HLT_HIMinBias*'))
-#
-    process.DQMStore.referenceFileName = '/dqmdata/dqm/reference/sistrip_reference_hi.root'
-    # Quality test for HI
-    process.stripQTester = cms.EDAnalyzer("QualityTester",
-                                     qtList = cms.untracked.FileInPath('DQM/SiStripMonitorClient/data/sistrip_qualitytest_config_heavyion.xml'),
-                                     prescaleFactor = cms.untracked.int32(3),
-                                     getQualityTestsFromFile = cms.untracked.bool(True),
-                                     qtestOnEndLumi = cms.untracked.bool(True),
-                                     qtestOnEndRun = cms.untracked.bool(True)
-                                     )
-
-    process.trackingQTester.qtList                  = cms.untracked.FileInPath('DQM/TrackingMonitorClient/data/tracking_qualitytest_config_heavyion.xml')
-    process.trackingQTester.prescaleFactor          = cms.untracked.int32(2)
+    if ((process.runType.getRunType() == process.runType.hi_run) and live):
+        process.source.SelectEvents = cms.untracked.vstring(
+            'HLT_HICentralityVeto*',
+            'HLT_HIMinBias*',
+            'HLT_HIZeroBias*'
```

Comment:
@icali, please clarify if we are going to have other HI tests in 10.3.X using cosmics-like menus

Reply:
No, next test should be with a quasi-final HI HLT menu. I let decide the DQM expert if you want add the extra paths here so that an unique HI DQM can be used for all the run mode (collisions HI + cosmics + circulating) or if we continue with the strategy of switching DQM configuration (via run_mode) passing from collisions HI and circulating+cosmics.